.travis.yml

```diff
@@ -2,12 +2,12 @@ sudo: false
 language: python
 python:
     - "2.7"
-    - "3.3"
     - "3.4"
     - "3.5"
+    - "3.6"
     - "pypy"
 install:
-    - pip install coverage pytest pytest-cov flake8
+    - pip install coverage pytest "pytest-cov<2.6.0" flake8
```

Comment:
Why was this required? Could you add a comment explaining it?

Reply:
It's the error from this run: https://travis-ci.org/logpy/logpy/jobs/499555791#L529

Reply:
Apparently, `pytest-cov` versions greater than the pinned one will require a higher `pytest` version.  As an alternative, we could try the newer `pytest`.

Reply:
Oh I see, so the version of pytest installed on the travis environment is too old for newer versions of pytest-cov. That makes sense. It's fine to leave it as-is.